dev-libs/skalibs/skalibs-2.7.0.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="skarnet.org general-purpose libraries"
+HOMEPAGE="https://www.skarnet.org/software/skalibs/"
+SRC_URI="https://www.skarnet.org/software/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="doc ipv6 static-libs"
+
+DEPEND=""
+RDEPEND=""
+
+HTML_DOCS="doc/*"
+
+src_prepare() {
+	default
+
+	# Remove QA warning about LDFLAGS addition
+	sed -i "s/tryldflag LDFLAGS_AUTO -Wl,--hash-style=both/:/" "${S}/configure" || die
+
+	# configure overrides gentoo's -fstack-protector default
+	sed -i "/^tryflag CFLAGS -fno-stack-protector$/d" "${S}/configure" || die
+}
+
+src_configure() {
+	econf \
+		--datadir=/etc \
+		--dynlibdir=/$(get_libdir) \
+		--libdir=/usr/$(get_libdir)/${PN} \
+		--sysdepdir=/usr/$(get_libdir)/${PN} \
+		--enable-clock \
+		--enable-shared \
+		$(use_enable static-libs static) \
+		$(use_enable ipv6)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```

Comment:
`default`

Reply:
You mean I should remove the src_install definition and the doc use flag? I guess that'd be okay considering the docs are only half a megabtye large and the other s6* packages don't have the flag either.

Reply:
I think the intention was 
```
src_install() {
        default

        use doc && einstalldocs
}
```

Reply:
In any case, I've already pushed a new version which uses the default src_install (and hence installs the docs unconditionally), and I'm inclined to keep it this way. :)